src/index.d.ts

```diff
@@ -55,9 +55,9 @@ export interface FlickityOptions {
     cellSelector?: string;
 
     /**
-     * Zero-based index of the initial selected cell.
+     * Zero-based index or selector string of the initial selected cell.
      */
-    initialIndex?: number;
+    initialIndex?: number | string | undefined;

```

Comment:
Undefined is not really necessary since it's optional

Reply:
yea but i copied these line from [@types/filickity](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/flickity/index.d.ts#L92-L95)

Reply:
That doesn't mean it's always correct 😅

Reply:
yea, your right ;))), correct is always correct.